supabase/migrations/20240313073422_init.sql

```diff
@@ -0,0 +1,1027 @@
+    "created" timestamp with time zone not null default now(),
+    "updated" timestamp with time zone,
+    "address" character(42),
+    "location_id" integer
+);
+
+
+alter table "public"."wallets" enable row level security;
+
+alter sequence "public"."credits_id_seq" owned by "public"."credits"."id";
+
+alter sequence "public"."location_id_seq1" owned by "public"."locations"."id";
+
+alter sequence "public"."new_access_id_seq" owned by "public"."access"."id";
+
+alter sequence "public"."new_debits_id_seq" owned by "public"."debits"."id";
+
+alter sequence "public"."new_logs_id_seq" owned by "public"."logs"."id";
+
+alter sequence "public"."new_partners_id_seq" owned by "public"."partners"."id";
+
+alter sequence "public"."new_permits_id_seq" owned by "public"."permits"."id";
+
+alter sequence "public"."new_tokens_id_seq" owned by "public"."tokens"."id";
+
+alter sequence "public"."new_tokens_network_seq" owned by "public"."tokens"."network";
+
+alter sequence "public"."new_users_id_seq" owned by "public"."users"."id";
+
+alter sequence "public"."new_wallets_id_seq" owned by "public"."wallets"."id";
+
+alter sequence "public"."settlements_id_seq" owned by "public"."settlements"."id";
+
+alter sequence "public"."unauthorized_label_changes_id_seq" owned by "public"."labels"."id";
+
+CREATE UNIQUE INDEX credits_pkey ON public.credits USING btree (id);
+
+CREATE UNIQUE INDEX location_node_id_node_type_key ON public.locations USING btree (node_id, node_type);
+
+CREATE UNIQUE INDEX location_pkey1 ON public.locations USING btree (id);
+
+CREATE UNIQUE INDEX location_unique_node ON public.locations USING btree (node_id, node_type);
+
+CREATE UNIQUE INDEX new_access_pkey ON public.access USING btree (id);
+
+CREATE UNIQUE INDEX new_debits_pkey ON public.debits USING btree (id);
+
+CREATE UNIQUE INDEX new_logs_pkey ON public.logs USING btree (id);
+
+CREATE UNIQUE INDEX new_partners_pkey ON public.partners USING btree (id);
+
+CREATE UNIQUE INDEX new_permits_pkey ON public.permits USING btree (id);
+
+CREATE UNIQUE INDEX new_tokens_pkey ON public.tokens USING btree (id);
+
+CREATE UNIQUE INDEX new_users_pkey ON public.users USING btree (id);
+
+CREATE UNIQUE INDEX new_wallets_pkey ON public.wallets USING btree (id);
+
+CREATE UNIQUE INDEX new_wallets_wallet_key ON public.wallets USING btree (address);
+
+CREATE UNIQUE INDEX partners_wallet_key ON public.partners USING btree (wallet_id);
+
+CREATE UNIQUE INDEX permits_nonce_key ON public.permits USING btree (nonce);
+
+CREATE UNIQUE INDEX permits_signature_key ON public.permits USING btree (signature);
+
+CREATE UNIQUE INDEX permits_transaction_key ON public.permits USING btree (transaction);
+
+CREATE UNIQUE INDEX settlements_pkey ON public.settlements USING btree (id);
+
+CREATE UNIQUE INDEX unauthorized_label_changes_pkey ON public.labels USING btree (id);
+
+alter table "public"."access" add constraint "new_access_pkey" PRIMARY KEY using index "new_access_pkey";
```

Comment:
also here it's much easier to read if primary keys and foreign keys are defined in the table

Reply:
I checked if SQL dump had options to prettify or change generation but it doesn't seem like it. Since I just diff / dumped the current db, I could change this but it would take me ages to go through each table one by one